akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -128,7 +128,7 @@ object GraphStageLogic {
   final class StageActor(
     materializer:     ActorMaterializer,
     getAsyncCallback: StageActorRef.Receive ⇒ AsyncCallback[(ActorRef, Any)],
-    initialReceive:   StageActorRef.Receive) {
+    createReceive:    StageActorRef.Receive) {
```

Comment:
Should this really have been renamed, its not a lambda but a receive that was already created, keep old name?